modality-probe-capi/README.md

```diff
@@ -335,17 +354,23 @@ syslog(
         now.event_count);
 ```
 
+This will place a Modality causal-coordinate into your log message, so
+that later in offline processing any given log message can be correlated
+with a specific location in the Modality probe's logical timeline. With
+just a little bit of effort you can now stitch together the causal history
```

Comment:
Replace "With just a little bit of effort" with instructions or remove that part from the sentence.